config/settings.yml

```diff
@@ -162,12 +147,6 @@
     :inventory_object_refresh: true
     :allow_targeted_refresh: true
     :refresh_interval: 15.minutes
-  :embedded_ansible_automation:
```

Comment:
Looks ok...I just removed entire sections related to embedded ansible, which seems to be done here.

Reply:
Is `:job_data_retention_days: 120` (above) in my original PR?  I vaguely recall leaving it in so that the purger could do its thing.

Reply:
I don't think that is directly related to the purger. This might actually be something that became unnecessary when the data was moved off of the filesystem on the tower side.

Some background...

The job data retention setting was added because the tower job metadata and event data was being stored on the filesystem. The request was to add this to log rotate, but as these were not logs this didn't really make sense. What this actually controls is a schedule which causes us to trigger a tower system job which cleans up job data older than a passed setting.

This is being configured and run in the EmbeddedAnsible class here:
https://github.com/ManageIQ/manageiq/blob/09a9f1a07d834008eb9415ff5ce49b4233d28838/lib/embedded_ansible.rb#L51-L65

All that said, I don't think we have to care about this setting as long as were not storing the ansible-runner events on the filesystem after the playbooks are run.

Reply:
Just for reference, your original PR:

https://github.com/ManageIQ/manageiq/pull/18657/files#diff-646ad6b10917e4385cbcc8c8524e24a2